zh.perl6intro.adoc

```diff
@@ -1369,10 +1369,9 @@ Type check failed for return value; expected Int but got Rat (1.44)
 
 [TIP]
 ====
-类型限制不仅控制返回值的类型，它也可以控制返回值是否定义。
-(Not only can type constraints control the type of the return value; they can also control its definedness.)
+类型限制不仅可以控制返回值的类型，还可以控制返回值是否需要严格定义。
 
-之前的例子中，我们指定了返回值必须为 `Int`，这与返回值的定义无关。
+之前的例子中，我们指定了返回值必须为 `Int`。 +
```

Comment:
I think we need "irrespective of its definedness", but use "定义状态(definedness)" rather than "定义(define)". It is hard for me to find a better translation for "definedness". Do you have some suggestions?